src/types/dispute-status-types.js

```diff
@@ -1,3 +1,2 @@
 export const DISPUTE_STATUS_OPEN = Symbol('DISPUTE_STATUS_OPEN')
```

Comment:
Should we move from using these to use `Status` in Disputes/types.js?

Reply:
[see here](https://github.com/aragon/court-dashboard/blob/1012dc82b449856e4557903a8fa12b35c4132705/src/types/types.js#L1) we re using this file now and is outside the Components/Disputes/Types